gdx-pay-android-googlebilling/src/com/badlogic/gdx/pay/android/googlebilling/PurchaseManagerGoogleBilling.java

```diff
@@ -29,6 +29,18 @@
     private PurchaseObserver observer;
     private PurchaseManagerConfig config;
 
+    /**
```

Comment:
Better move this JavaDoc to the setter methods.

Reply:
Done!